src/atecc/atecc.h

```diff
@@ -33,94 +36,20 @@ typedef enum {
     ATECC_ERR_INVALID_ARGS = -8,
 } atecc_error_t;
```

Comment:
I will probably unify this and optiga_error_t into a single securechip_error_t again, here or in another PR.

Reply:
Done now.